README.md

```diff
@@ -1,2 +1,150 @@
+image.
+
+You may wish to use the image locally, or push it to a suitable private container repository. Please
+note that the Parsec maintenance team will ensure that the `hello-parsec` container images are
+available in a suitable public location.
+
+# How to Run
+
+This container is designed to check a system where the Parsec service is running. If you do not have
+Parsec running yet, you might want to view the [quickstart
+guide](https://parallaxsecond.github.io/parsec-book/getting_started/index.html) to get started with
+the service.
+
+Parsec clients talk to the service using a Unix domain socket. When the client application is
+running in a container, as is the case here, the container needs to be able to see the domain socket
+on the host. This can be achieved by running the Docker image with a [bind
+mount](https://docs.docker.com/storage/bind-mounts/). By default, Parsec expects to find the domain
+socket file in the folder `/run/parsec`. If this is the folder that your service is using (which,
+again, will be the case unless you have explicitly configured it to do otherwise), then the bind
+mount is very simple:
+
+```
+docker run -v /run/parsec:/run/parsec hello-parsec
+```
+
+This command will execute the demo client application.
+
+# What Should I See?
+
+If the `hello-parsec` demo container application successfully connects with the service, you should
+see the banner message, followed by some details of the service configuration, and finally the
+results of the RSA encryption round-trips using the example programs in their different languages.
+
+The output should be similar to what is shown below. It may vary slightly depending on your service
+configuration. If you do not see the Parsec banner, then something has gone wrong, and you will need
+to ensure that Parsec is running correctly on your system, and that you have supplied the correct
+bind mount. Make sure, for example, that the file `/run/parsec/parsec.sock` exists, or that you have
+adjusted the above command with the correct path where `parsec.sock` can be found.
+
+```
+Checking for availability of the Parsec service on your system... 
+
+                             o   o----------------
+                             |  --------o     o  |
+                             |  |  ---------  |  |
+                             |  |  |       |  |  |
+                             |  |  |       |  |  |
+                             |  |  |       |  |  |
+                             |  |   \      o  |  |
+                             \  o    \        /  /
+                              \   o   \      /  /
+                               \   \   o    /  o
+                                \   \      /  o
+                                 \   \    /  /
+                                  \   \  /  /
+                                   ---------
+                                    -------
+
+
++++++++          +          +++++++         ++++++       ++++++++        +++++   
```

Comment:
nit: trailing white spaces after `+++++`

Reply:
Thanks, didn't spot that. I can fix.